src/arrays.js

```diff
@@ -1,61 +1,107 @@
 const getNthElement = (index, array) => {
   // your code here
+  if (array.length > index) {
+    return array[index];
+  }
+  return array[index - array.length];
 };
 
 const arrayToCSVString = (array) => {
   // your code here
+  return array.toString();
 };
 
 const csvStringToArray = (string) => {
   // your code here
+  const str = string.split(',');
+  return str;
 };
 
 const addToArray = (element, array) => {
   // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
   // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
   // your code here
+
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
   // your code here
+  return numbers.join().split(',');
 };
 
 const uppercaseWordsInArray = (strings) => {
   // your code here
+ // return strings.map(text => text.toUpperCase());
+
+ for (let i = 0; i < strings.length; i++) {
+     strings[i] = strings[i].toUpperCase();
+   }
+   return strings;
 };
 
 const reverseWordsInArray = (strings) => {
   // your code here
+  const newArr = [];
+  for (let i = 0; i < strings.length; i++) {
+    const newString = strings[i].split('').reverse().join('');
+    newArr.push(newString);
+  }
+  return newArr;
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+  // your 
+  
+  const even = [];
+  for (let i = 0; i < numbers.length; i++) {
+    if ((numbers[i] % 2) === 0) {
+      even.push(numbers[i]);
+    }
+  }
+  return even;
 };
 
 const removeNthElement2 = (index, array) => {
   // your code here
+  return array.filter((_, itemIndex) => itemIndex !== index);
 };
 
 const elementsStartingWithAVowel = (strings) => {
   // your code here
+  return strings.filter(word => ['a','e','i','o','u'].includes(word[0].toLowerCase()));
 };
 
 const removeSpaces = (string) => {
   // your code here
+  return string.split(' ').join('');
 };
 
 const sumNumbers = (numbers) => {
   // your code here
+  let sum = 0;
+  for (i = 0; i < numbers.length; i++) {
+    sum += numbers[i];
+  }
+  return sum;
 };
 
 const sortByLastLetter = (strings) => {
```

Comment:
The test is failing on this one, a couple of issues I can pinpoint are:

- using a `for loop` is not wrong per se but we prefer to use Array methods available to us that make our code more explicit. In this case, we could have used a `forEach`  or `map`.
💡 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
💡 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

- the `sort` method on an array takes a function and not a single value (which is what `strings[i].charAt(strings[i].length - 1)` evaluates to. 
💡 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

- the value of performing the operations in `strings.sort(...));` is being stored nowhere. We can store it in its own variable like `const result = strings.sort(...) which then we can push to the `sorted` array. As it stands, pushing `trings[i]` is pushing the current element of the array being iterated without any transformation.

- code should be indented correctly for organization and ease of reading (note line 101 and 102)